### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by Mathadon**
This is for #575 

@GlennReynders or @rubenbaetens can you review and/or accept these changes?

### CODE REVIEW

File: IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/InteriorConvectionWindow.mo [hunk 1]
```diff
@@ -1,68 +0,0 @@
-within IDEAS.Buildings.Components.BaseClasses.ConvectiveHeatTransfer;
```
Comment (GlennReynders) [inline IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/InteriorConvectionWindow.mo]: @Mathadon  did you mean to delete this model?

Comment (GlennReynders) [inline IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/InteriorConvectionWindow.mo]: Did you mean to delete this model?

**Comment by Mathadon** [comment]
unit tests are ok

**Comment by Mathadon** [comment]
@GlennReynders this can be merged

**Comment by rubenbaetens** [comment]
Resolves #575.

**Pull request merged**